lib/SILOptimizer/Utils/Existential.cpp

```diff
@@ -195,6 +211,9 @@ static SILValue getAddressOfStackInit(SILValue allocStackAddr,
   if (auto *CAI = dyn_cast<CopyAddrInst>(initI))
     return CAI->getSrc();
 
```

Comment:
`getAddressOfStackInit` doesn't make sense any more as an API. I think it should just be inlined into `OpenedArchetypeInfo`'s ctor since it seems entirely specific to that context. The the nonsense API name and invalid comments can just be deleted.

Reply:
Agreed. That's much better. Thanks.